src/evaluationSummaryAggregator.ts

```diff
@@ -0,0 +1,119 @@
+// TODO: flush on beforeunload
+
+// TODO: should we retry the data chunk if a flush fails?
+
+// TODO: pause when offline?
+
+// TODO: flush when we receive a config update (or as a result of a context update...but that should trigger a config update anyway)
+
+// TODO: client option to disable telemetry?
```

Comment:
@semanticart would love your opinion on importance of these TODOs

Reply:
I think the option to disable telemetry is the only one of these I'd consider a blocker (for releasing a new version, not necessarily merge)

Reply:
I think telemetry should be opt-in for the initial release. This will allow us to monitor it better in real-world scenarios before making it on-by-default for others.